error_test.go

```diff
@@ -43,11 +43,16 @@ func (err *errTest) Error() string {
 
 // Format implements fmt.Formatter.
 func (err *errTest) Format(f fmt.State, c rune) {
-	if !f.Flag('+') || c != 'v' {
+	if c != 'v' {
 		fmt.Fprint(f, "unexpected verb for formatting the error")
 	}
 	fmt.Fprint(f, err.Error())
 	if err.formatted {
 		fmt.Fprint(f, "\n  file:line")
 	}
 }
+
+func errTestPtr() **errTest {
```

Comment:
Why not just use `new(*errTest)` inline in the tests?

Reply:
Huh. Only because it didn't occur to me!

Updated.